src/main/java/songbox/house/service/impl/GoogleAuthenticationServiceImpl.java

```diff
@@ -51,6 +49,16 @@
     private static final GoogleClientSecrets GOOGLE_CLIENT_SECRETS = loadClientSecrets();
     private static final GoogleAuthorizationCodeFlow GOOGLE_AUTHORIZATION_CODE_FLOW = initFlow();
 
+    @Autowired
+    public GoogleAuthenticationServiceImpl(
+            @Value("${songbox.house.google.auth.redirectdomain}") String redirectDomain,
```

Comment:
For REST project this prop is dynamic(it depends on request), but you made it static